src/actions/PushOperation.ts

```diff
@@ -134,6 +138,8 @@ export class PushOperation {
         });
 
         this._assetKind = this.assetDefinitions[0].kind;
+
+        this._baseKind = await this.resolveBaseKind();
```

Comment:
I wanted to ask you about line 140, is it ok that we pick the first assetDefinition? Wondering because a `kapeta.yml` file can describe multiple assets, right?